quantum/split_common/transport.c

```diff
@@ -55,7 +55,7 @@ bool transport_master(matrix_row_t matrix[]) {
 
   // write backlight info
 #  ifdef BACKLIGHT_ENABLE
-  uint8_t level = get_backlight_level();
+  uint8_t level = backlight_config.enable ? backlight_config.level : 0;
```

Comment:
This should probably be using the equivalent functions, just in case the underlying struct changes at some point:

```suggestion
  uint8_t level = is_backlight_enabled() ? get_backlight_level() : 0;
```

Reply:
Agreed. Pushed an updated patch that removes the direct access for i2c and serial.